src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -445,6 +445,23 @@ You can set any link:http://docs.oracle.com/javase/8/docs/api/java/io/Serializab
 http://docs.oracle.com/javase/8/docs/api/java/lang/Object.html#toString()[toString()^]
  method when required, for instance, to make those values available as environment variables in native tasks.
 
+==== Task variables
+
+In a task you can define *variables* that are available only in the scope of that task. Therefore, those variables won't be propagated to depending tasks. However, if a different value is set into a script, this value will be propagated. A variable is defined using the following attributes:
+
+* *name*: the name of the variable
+* *value*: the value of the variable
+* *inherited*: asserts whether the propagated value should be use instead of the *value* value. If *true*, the *value* field can be empty. As a consequence, inheritance requires to declare the variable value at workflow level
```

Comment:
I think that you should not use * around "true". If "true" is not a variable, it can be confusing, here the pattern *variable* can be a variable or a value. I think that you should use different patterns for variables and values.